src/index.ts

```diff
@@ -0,0 +1,20 @@
+import {
+  JupyterLab, JupyterLabPlugin
+} from '@jupyterlab/application';
+
+import '../style/index.css';
```

Comment:
I don't think you can generally import a CSS file into a typescript file. Separately, I'm not sure how you'll get that into the extension.

Reply:
(Either way, we'll find out when I try to use this in a follow-up PR.)